packages/dd-trace/src/profiling/profiler.js

```diff
@@ -82,7 +83,7 @@ class Profiler extends EventEmitter {
 
   _nearOOMExport (profileType, encodedProfile) {
     const start = this._lastStart
-    const end = new Date()
+    const end = Date.now()
```

Comment:
Is this okay? As far as I can tell, we use `Date` objects for start and end throughout. We _could_ consistently use millis everywhere, but that'd be a larger change.

Reply:
I messed up my commits, fixed.